app/views/pages/connect.html.erb

```diff
@@ -16,7 +19,8 @@
   	<ul class="vcard connect-info">
   		<li><i class="fa fa-home">&nbsp;</i>13 Computing Drive, Computing 1, #02-14, Singapore 117417</li>
   		<li><i class="fa fa-phone">&nbsp;</i>+65 6516 6759</li>
-  		<li><i class="fa fa-envelope-o">&nbsp;</i>connect@nuscomputing.com</li>
+  		<!-- go to app/assets/javascripts/connect.js to update the contact email -->
+  		<li><i class="fa fa-envelope-o">&nbsp;</i><span class="connect-email"></span></li>
```

Comment:
why does this need to be dynamic? if client disables javascript, then the email won't show up